internal/cmd/testdata/edgecase/generator_config.yml

```diff
@@ -1,6 +1,9 @@
 provider:
   name: edgecase
   schema_ref: '#/components/schemas/edgecase_provider'
+  ignores:
```

Comment:
Provider generator config looks a little different compared to the resources/data sources, but it didn't make as much sense to have the same structure 🤷🏻 

Open to suggestions